cmd/tiller-proxy/main.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright (c) 2018 Bitnami
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/heptiolabs/healthcheck"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/pflag"
+	"github.com/urfave/negroni"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/helm/pkg/chartutil"
+	"k8s.io/helm/pkg/helm"
+	"k8s.io/helm/pkg/helm/environment"
+
+	"github.com/kubeapps/kubeapps/cmd/tiller-proxy/proxy"
+)
+
+var (
+	settings environment.EnvSettings
+	p        *proxy.Proxy
```

Comment:
since this is a global var, can we call it `proxy` to make it more obvious what it is?